bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,200 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="tapocontrol"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- ############################### BRIDGE ############################### -->
+	<bridge-type id="bridge">
+		<label>TapoControl Cloud-Login</label>
+		<description>TapoControl Cloud Connector</description>
+		<config-description-ref uri="bridge-type:tapo:bridge"/>
+	</bridge-type>
+
+
+	<!-- ############################### DEVICES ############################### -->
+	<!-- P100 THING-TYPE (SOCKET) -->
+	<thing-type id="P100">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+
+		<label>Tapo P100 SmartPlug</label>
+		<description>Tapo Plug device with relay</description>
+		<channel-groups>
+			<channel-group id="actuator" typeId="smartPlug"/>
+			<channel-group id="device" typeId="deviceState"/>
+		</channel-groups>
+		<representation-property>deviceName</representation-property>
+
+		<config-description-ref uri="thing-type:tapo:device"/>
+	</thing-type>
+
+	<!-- L510E THING-TYPE (WHITE-LIGHT-BULB) -->
+	<thing-type id="L510_Series">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+
+		<label>Tapo L510 Series White-Light-Bulb</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions